packages/dev/docs/pages/Updates.mdx

```diff
@@ -10,4 +10,4 @@ governing permissions and limitations under the License. -->
 import {Layout} from '@react-spectrum/docs';
 export default Layout;
 
-# Announcements
+# Updates
```

Comment:
Updates?

what do we expect to live here?

Reply:
I imagined it would point to https://github.com/adobe-private/react-spectrum-v3/releases, and/or more general updates? Kind of like when we share updates in Slack?